### DISCUSSION TITLE
What do you think about a GLFW backend?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/eepp#66] [General]
Don't you think SDL is too bloated?

**Comment by SpartanJ** [comment]
No, I don't think so, but it's a much more complete library than GLFW. I did implement an GLFW backend once but it's so feature limited that I dropped it even before pushing it to the repo. What I do care is about having the features I need to provide a good desktop application experience, GLFW is more oriented for games, it does not cover my needs. It's true that SDL2 provides some stuff I don't particularly use (the rendering stuff), but SDL2 builds in a few seconds, it's still a small library, extremely portable and polished, currently maintained basically by Valve with a very open license (MIT).
